NppJSONViewer/NppJsonViewer/resource.rc

```diff
@@ -105,6 +105,7 @@ BEGIN
     CONTROL         "Ignore trailing comma",IDC_CHK_IGNORE_COMMA,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,5,33,140,10
     CONTROL         "Ignore comments in json",IDC_CHK_IGNORE_COMMENT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,5,46,140,10
     CONTROL         "Use json highlighting",IDC_CHK_JSON_HIGHLIGHT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,5,59,140,10
+    CONTROL         "Replace value 'undefined' with 'null'",IDC_CHK_REPLACE_UNDEFINED,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,5,59,140,10
```

Comment:
Unfortunately, it overlaps with previous control. You need to change `59` to `73`.

Reply:
I've overlooked there's a new option in the meantime, made it the same distance like the others.